packages/form-plugin/src/directive.ts

```diff
@@ -20,7 +20,15 @@ export class FormDirective implements OnInit, OnDestroy {
   debounce = 100;
 
   @Input('ngxsFormClearOnDestroy')
-  clearDestroy = false;
+  set clearDestroy(val: boolean) {
```

Comment:
`val: boolean | string`

Reply:
@splincode I wrote  the set getter so type issue for getter I think boolean is right here or any?

Reply:
I think that it could stay as boolean, unless there is a type issue?